powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/model/request/VerifyOfflineSignatureRequest.java

```diff
@@ -34,11 +35,23 @@
 @Data
 public class VerifyOfflineSignatureRequest {
 
+    @Schema(description = "Activation identifier")
+    @NotBlank(message = "Activation ID must not be empty when verifying offline signature")
     private String activationId;
+
+    @Schema(description = "Signed data")
+    @NotNull(message = "Parameter data must not be null when verifying signature")
     private String data;
+
+    @Schema(description = "Signature")
+    @NotBlank(message = "Signature must not be empty when verifying signature")
     @ToString.Exclude
     private String signature;
+
+    @Schema(description = "Signature component length")
     private BigInteger componentLength;
+
+    @Schema(description = "Whether biometric factor is allowed")
```

Comment:
I am not sure about this one, just suggesting to keep the naming consistent.
```suggestion
    @Schema(description = "Whether biometry signature type is allowed")
```

Reply:
It's a factor, not signature type. Signature type is `POSSESSION_BIOMETRY`.